iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HvdcLineAdderImpl.java

```diff
@@ -116,12 +116,12 @@ public HvdcLine add() {
         }
         VoltageLevelExt vl1 = converterStation1.getTerminal().getVoltageLevel();
         VoltageLevelExt vl2 = converterStation2.getTerminal().getVoltageLevel();
-        if (vl1.getClosestNetwork() != vl2.getClosestNetwork()) {
```

Comment:
Why do you need these two messages? what is the difference of the two if? And do we really want to throw at element creation? If no detach, no problem?